### PR TITLE
Feature/toolbar

### PR DESCRIPTION
**Pull request by hervecarol**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
An application toolbar was added.

## Fixes
Link all the related issues from the issue tracker
- Closes: #

## How has this been tested ?
(If possible) describe the steps to test these modifications.

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/src/components/LabelAndValueModule.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './switch.css';
+
+
+
+const LabelAndValueModule = (props: any) => {
```
Comment (laurentboisvert) [inline src/src/components/LabelAndValueModule.tsx:6]: If these modules are only to be consumed by the toolbar, I would do a module for the Toolbar and set its child components under it:
```
toolbarmodule
    ToolbarModule.tsx
    modules
      LabelAndValueModule.tsx
      BatteryLevelIndicatorModule.tsx
```
Thoughts?

**Review by laurentboisvert — CHANGES_REQUESTED** [review]

File: src/src/components/LabelAndValueModule.tsx (hunk 1 above)
Comment (hervecarol) [reply src/src/components/LabelAndValueModule.tsx:6]: The `LabelAndValueModule` is a generic module that can be used in any other module.

File: src/src/App.tsx [hunk 2]
```diff
@@ -7,64 +7,148 @@ import ImageViewer from "./components/ImageViewer";
...
-                                <Thruster key={id}
-                                    effort={thruster.effort}
-                                    identification={thruster.ID}
-                                    minMark={-100}
-                                    maxMark={100}
-                                    step={25}
-                                    thumbEnabled={isDryRunMode} />
-                            )
-                        })}
+                        <Thruster key={1}
```
[the remark this answers is not in the document]
Comment (hervecarol) [reply src/src/App.tsx:85]: Yes it was intended. I had to make some changes to the thrusters because of the format of the thrusters data.

**Review by antho6222 — APPROVED** [review]

File: src/src/components/BatteryLevelIndicatorModule.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import Slider from "@material-ui/core/Slider";
+import { withStyles } from '@material-ui/core/styles';
+import LabelAndValueModule from "./LabelAndValueModule";
+
+const ThrusterEffortIndicator = withStyles({
```
[the remark this answers is not in the document]
Comment (hervecarol) [reply src/src/components/BatteryLevelIndicatorModule.tsx:6]: Yes it was an error. it is fixed now.

**Pull request merged**